travis-bootstrap.sh

```diff
@@ -42,11 +42,11 @@ timed cabal update
 # issue, and it is easy enough to work around: clean first.
 
 echo Cabal
-(cd Cabal && timed cabal clean)
```

Comment:
Well, you spent enough time on this already, so let's merge this patch in the current form. In the (distant) future maybe we can rewrite these scripts and `bootstrap.sh` in Haskell.